src/main/java/nextstep/subway/domain/Path.java

```diff
@@ -17,7 +17,17 @@ public int extractDistance() {
         return sections.totalDistance();
     }
 
+    public int extractDuration() {
+        return sections.totalDuration();
+    }
+
     public List<Station> getStations() {
         return sections.getStations();
     }
+
+    public int extractFare() {
+        int fare = Fare.calculate(extractDistance());
+        return fare;
```

Comment:
개인성향 차이가 있지만 가독성에 지장이 없는 경우에는 라인을 줄이는 것도 괜찮은 것 같습니다 😄 

```suggestion
        return Fare.calculate(extractDistance());
```